### PR TITLE
doc: Documentation review for Readme and AWS docs

### PR DESCRIPTION
**Pull request by umapraseeda**
Documentation review for Readme and AWS docs

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Comment by coderbyheart** [comment]
Most errors are due to hyperlink targets not being referenced. Is that allowed in the NCS docs? If yes, I'll see if I can suppress this type of error, otherwise I think it's OK to remove them, because they are not used after all.

File: docs/aws/Authentication.rst [hunk 1]
```diff
@@ -1,49 +1,38 @@
-================================================================================
+.. _authentication:
+
 Authentication
-================================================================================
+##############
 
-Web App
-================================================================================
+The `Cat Tracker Web Application <https://bifravst.github.io/bifravst/docs/app/GettingStarted.html>`_ on AWS connects to the `AWS IoT broker <https://docs.aws.amazon.com/iot/latest/developerguide/iot-connect-devices.html>`_ using WebSockets, and the authentication is done through `AWS Cognito <https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html>`_.
```
Comment (coderbyheart) [inline docs/aws/Authentication.rst]: Do not use absolute links here, we will move the docs to https://developer.nordicsemi.com/ in the future (see #56), and this should be doable without changing all links.

This also makes any preview link (e.g. from the downloaded ZIP file) not link to the file in the preview.
```suggestion
The `Cat Tracker Web Application <../app/AWS.html>`_ for AWS connects to the `AWS IoT broker <https://docs.aws.amazon.com/iot/latest/developerguide/iot-connect-devices.html>`_ using WebSockets, and the authentication is done through `AWS Cognito <https://docs.aws.amazon.com/cognito/latest/developerguide/what-is-amazon-cognito.html>`_.
```

File: docs/aws/ContinuousDeployment.rst [hunk 2]
```diff
@@ -1,71 +1,62 @@
-================================================================================
-Continuous Deployment
-================================================================================
+.. _continuous_dep:
+
+Continuous deployment
+#####################
+
+You can enable continuous synchronization of your deployment with the source repository.
```
Comment (coderbyheart) [inline docs/aws/ContinuousDeployment.rst]:
```suggestion
You can enable continuous deployment of the changes in your source repository to an AWS account.
```

File: README.rst [hunk 3]
```diff
@@ -31,88 +34,82 @@ Bifravst
...
-- *How can you develop a cellular IoT product that maximizes battery life,
-  minimizes data usage, and handles unreliable connectivity gracefully?*
+* How can you connect Nordic's cellular IoT chips to your cloud provider?
+* How do devices send data into the cloud?*
+* How can the data be sent to the devices?
+* How can users and other services interact with the devices?
+* How can you upgrade the application firmware of your devices while they are deployed in the field?
+* How can you develop a cellular IoT product that maximizes battery life, minimizes data usage, and handles unreliable connectivity gracefully?
 
-*Bifravst* is licensed under the `3-clause BSD license <./LICENSE>`_ (software)
```
Comment (coderbyheart) [inline README.rst:68]: What is the reason for remove the license section?

File: docs/aws/ContinuousIntegration.rst [hunk 4]
```diff
@@ -1,72 +1,59 @@
...
-You can run these test when developing:
+You can run the following test during development:
 
 .. code-block:: bash
 
     npm run test:e2e
 
-Set up on GitHub
-================================================================================
+Setting up the AWS account and providing the variables in GitHub
```
Comment (coderbyheart) [inline docs/aws/ContinuousIntegration.rst]:
```suggestion
Setting up the AWS account and providing the environment variables in GitHub
```

File: docs/aws/UnwiredLabsAPI.rst [hunk 5]
```diff
@@ -1,30 +1,22 @@
...
-================================================================================
+.. _unwired_labs_api:
 
-.. note::
+Unwired Labs API
+################
 
-    This enables the `cell geolocation lookup <../app/CellGeolocation.html>`_
-    for the app using the Unwired Labs API.
+You can enable the `cell geolocation lookup <https://bifravst.github.io/bifravst/docs/app/CellGeolocation.html>`_ for the app using `Unwired Labs <https://unwiredlabs.com/>`_ geolocation API in your deployment.
```
Comment (coderbyheart) [inline docs/aws/UnwiredLabsAPI.rst]: Again, do not use absolute links to `bifravst.github.io`.
```suggestion
You can enable the `cell geolocation lookup <../app/CellGeolocation.html>`_ for the app using `Unwired Labs <https://unwiredlabs.com/>`_ geolocation API in your deployment.
```

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Comment by umapraseeda** [comment]
> Most errors are due to hyperlink targets not being referenced. Is that allowed in the NCS docs? If yes, I'll see if I can suppress this type of error, otherwise I think it's OK to remove them, because they are not used after all.

Yes, such targets that serve as IDs to the docs are allowed in NCS even if they are not in use.

**Comment by coderbyheart** [comment]
Ok, I've added a config that ignores these errors in 76adfd65a9bc952bcd6a479d7dce547d7b1e9208

File: README.rst (hunk 3 above)
Comment (umapraseeda) [reply README.rst:68]: I shall fix this. I tried to move this sentence, but it might have accidentally got deleted. Thanks for spotting this!

**Comment by coderbyheart** [comment]
Why are some links using ref and others link using the filename?

https://github.com/bifravst/bifravst/pull/229/files#diff-8b1b67da1591810d2e625580322e807fbb0201aa2dc751063d32ba676ca1dae9R117

```rst
You can now `deploy the web application <../app/AWS.html>`_ and :ref:`provision credentials to your devices <device_cred>`.
```

**Comment by coderbyheart** [comment]
A few issues remain:

```
./README.rst:4: (INFO/1) Duplicate implicit target name: "bifravst".
./docs/aws/SNI.rst:4: (INFO/1) Duplicate implicit target name: "sni".
./docs/aws/Authentication.rst:4: (INFO/1) Duplicate implicit target name: "authentication".
```

**Comment by umapraseeda** [comment]
> Why are some links using ref and others link using the filename?
> 
> https://github.com/bifravst/bifravst/pull/229/files#diff-8b1b67da1591810d2e625580322e807fbb0201aa2dc751063d32ba676ca1dae9R117
> 
> ```
> You can now `deploy the web application <../app/AWS.html>`_ and :ref:`provision credentials to your devices <device_cred>`.
> ```

I am planning to update this link to ref, after I start adding the IDs to the documents under Cat Tracker web application. All such remaining links (<../app/AWS.html>) will be updated in the upcoming PR for Cat Tracker web application.

**Pull request merged**